frontend/src/screens/channels/CurrentChannelOrder.tsx

```diff
@@ -111,29 +109,17 @@ function ChannelOrderInternal({ order }: { order: NewChannelOrder }) {
 
 function Success() {
   return (
-    <div className="flex flex-col justify-center gap-5 p-5 max-w-md items-stretch">
-      <TwoColumnLayoutHeader
-        title="Channel Opened"
-        description="Your new lightning channel is ready to use"
-      />
+    <div className="w-80 flex flex-col gap-6 px-8 py-6 items-center justify-center text-secondary-foreground border rounded-xl">
+      <p className="font-medium">New channel is ready!</p>
 
-      <p>
-        Congratulations! Your channel is active and can be used to send and
-        receive payments.
-      </p>
-      <p>
-        To ensure you can both send and receive, make sure to balance your{" "}
-        <ExternalLink
-          to="https://guides.getalby.com/user-guide/v/alby-account-and-browser-extension/alby-hub/liquidity"
-          className="underline"
-        >
-          channel's liquidity
-        </ExternalLink>
-        .
+      <img src={confirmation} className="w-52 h-52" />
+
+      <p className="text-center">
+        Your new channel is open and ready for payments.
       </p>
 
-      <Link to="/home" className="flex justify-center mt-8">
```

Comment:
after channel being opened we go to /home so that user can complete steps present on home page? is it fine that we got to /channels? if not i will revert route and copy back to existing one